app-text/typst/typst-0.13.1.recipe

```diff
@@ -11,13 +12,13 @@ Typst has:
 HOMEPAGE="https://github.com/typst/typst"
 COPYRIGHT="2022 Laurenz Mädje and Martin Haug."
 LICENSE="Apache v2"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
 CHECKSUM_SHA256="2ffd8443668bc0adb59e9893f7904fd9f64dce8799a1930569f56a91305e8b71"
 PATCHES="typst-$portVersion.patchset"
 
-ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="x86"
+ARCHITECTURES="x86_64 !x86_gcc2"
```

Comment:
```suggestion
ARCHITECTURES="x86_64 !x86_gcc2"
ARCHITECTURES="?all x86_64"
```

Reply:
This has only been tested on 64bit, so just mark all as untested (this includes x86_gcc2)

Reply:
Indeed. Thanks.